scripts/cs4-migration/all-ns-upgrade-ics.sh

```diff
@@ -309,48 +316,48 @@ echo "==> Starting IBM Cloud Pak foundational services upgrade..."
 
 COMMON_SERVICES_NAMESPACE=$(oc get operandrequest -n $WSA_INSTANCE_NAMESPACE websphereauto -o yaml | grep "registryNamespace: " | cut -d ":" -f2 | tr -d " ")
 if [[ "$COMMON_SERVICES_NAMESPACE" == "$WSA_OPERATOR_NAMESPACE" ]]; then
-  echo "The IBM Cloud Pak foundational services has already been upgraded! Exiting."
-  exit 1
-fi
-echo "IBM Cloud Pak foundational services namespace: $COMMON_SERVICES_NAMESPACE"
+  echo "The IBM Cloud Pak foundational services has already been upgraded so a new Subscription will not be applied."
```

Comment:
In the past when CS namespace was changed to operator namespace it would exit 1 here, but the script should yield for line 363 waiting for zen service to start because it could be still in the process of upgrading.